index.bs

```diff
@@ -1650,6 +1764,45 @@ SIGN).</p>
 
 <p>When interpreted as a number, a <a>WebVTT percentage</a> must be in the range 0..100.</p>
 
+<p>A <dfn>WebVTT attributes block</dfn> consists of the following components, in the given order:</p>
+<ol>
+ <li>The string "<code>ATTRIBUTES</code>".</li>
+ <li>
+  The following components, in the given order:
+  <ol>
+   <li>A <a>WebVTT line terminator</a>.</li>
```

Comment:
I am not sure, but this item seems to be out from this `<ol>`, since it divides the `ATTRIBUTES` line but not in key/value pairs

Reply:
I intended that as the EOL char on the same line as ATTRIBUTES

```
ATTRIBUTES
          ^
```

But I'd be happy to combine it with the previous line, or specify some other way at the editors' direction.